Resources/Prototypes/Entities/Objects/Weapons/Melee/mining.yml

```
@@ -11,22 +11,24 @@
     sprite: Objects/Weapons/Melee/pickaxe.rsi
     state: pickaxe
   - type: MeleeWeapon
-    attackRate: 0.7
+    attackRate: 0.75
     wideAnimationRotation: -135
     soundHit:
       path: "/Audio/Weapons/smash.ogg"
       params:
         volume: -3
     damage:
       groups:
-        Brute: 5
+        Blunt: 5
+        Piercing: 5
+        Structural: 10
   - type: Wieldable
   - type: IncreaseDamageOnWield
     damage:
       groups:
-        Brute: 10
-      types:
-        Structural: 30
+        Blunt: 15
```

What's failing the linter here is that you are attempting to load a damage *type* as a damage *group*
```suggestion
       types:
        Blunt: 15
        Piercing: 5
        Structural: 25
```